status-im/status-react
# fix #1486 chat: removable console chat

[comment]: # (Please replace ... with your information. Remove < and >)
[comment]: # (To auto-close issue on merge, please insert the related issue number after # i.e fixes #566)
fixes #1486 

### Summary:
[comment]: # (Summarise the problem and how the pull request solves it)
Console chat was previously removable, this shouldn't be the case and is fixed by this PR. 

### Steps to test:
- Open Status
- Go to Chats screen
- Select "Edit chats" from the  top bar menu
- Observe that Console chat can't be removed

[comment]: # (PRs will only be accepted if squashed into single commit.)
status: ready
